src/content/pages/remote.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: EuroPython 2024 Remote
+subtitle: Join EuroPython 2024 remotely!
+social_card: social_card_remote.png
+---
+
+# EuroPython 2024 Remote
+
+As happy as we are about finally being able to see many friends in **Prague**
+this year, we are also aware that many of you will not be able to make it in-person.
+Again, we present you EuroPython 2024 remote:
+Wherever you are based, let's celebrate Python and our community, rejoice at our diversity,
+learn and have fun in Prague and beyond! In this spirit, we strive to make the remote participation
+as interactive as possible.
+
+### How to participate in the conference remotely?
+
+- To participate in the conference interactively (with guaranteed access to livestreams), a "remote ticket" registration is required. See details and pricing on the [Tickets information](/tickets#remote-tickets) page;
+- All ticket holders will receive a link to join our Discord server for the conference.
+
+*Note: in-person ticket holders will be provided with remote event access as well.*
+
+### What is included in the remote ticket?
+Remote participation between 10-12 July (Wednesday - Friday):
+- Livestreams of Keynotes, Panel discussions, all 6 tracks of talks.
+- Live participation (text-based) on Q&A session post-talks.
+- Text-based discussions with available speakers.
+- Online job board and chat with sponsors at the remote booths.
+- On-Demand viewing, which will be available hours after the live stream.
+
+*Note: Tutorials and Workshops on 8 and 9 July will not be streamed.*
+
+### What changes ###
+This year, we will **NOT** provide open and free access to the livestreams of our conference talks (10-12 July). If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.
```

Comment:
... purchasing a remote ticket. Purchasing a remote ticket not only gives you access to the conference but also helps support the community!

Reply:
```suggestion
This year, we will **NOT** provide open and free access to the livestreams of our conference talks (10-12 July).
If you would like to watch EuroPython remotely, please consider purchasing a remote ticket.
Purchasing a remote ticket not only gives you access to the conference but also helps support the community!
```

Reply:
@laisbsc Is this suggestion OK to commit?